examples/compose/plot_column_transformer_mixed_types.py

```diff
@@ -75,7 +75,12 @@
 )
 
 categorical_features = ["embarked", "sex", "pclass"]
-categorical_transformer = OneHotEncoder(handle_unknown="ignore")
+categorical_transformer = Pipeline(
```

Comment:
As mentioned https://github.com/scikit-learn/scikit-learn/issues/23053#issuecomment-1089253161, I think we should update the docstring to say that `OneHotEncoder` will treat the missing category as it's own category.

Reply:
More specifically, I think we can update:

https://github.com/scikit-learn/scikit-learn/blob/582fa30a31ffd1d2afc6325ec3506418e35b88c2/examples/compose/plot_column_transformer_mixed_types.py#L14-L16

Reply:
I couldn't see where it actually labels the column with the string 'missing', does it for sure do that? I just pushed a commit modifying the doc string.

Reply:
It probably gets too complicated for the doc string update, but the OneHotEncoder docs implied that different kinds of missing values would get their own labels. Like in your example in the issue with `get_feature_names_out`, the label was `nan`, and there would be another column yet if the values were `None` instead of `np.nan`. Testing.